releasenotes/notes/add-backend-estimator-v2-26cf14a3612bb81a.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    The implementation :class:`~.BackendEstimatorV2` of :class:`~.BaseEstimatorV2` was added.
```

Comment:
can an example like https://github.com/Qiskit/qiskit/pull/11931#issuecomment-1988263157 be added here?